src/core/tests/type_prop/reshape.cpp

```diff
@@ -20,6 +20,18 @@ TEST(type_prop, static_value_propagation) {
     ASSERT_EQ(r->get_shape(), (Shape{1, 2, 3}));
 }
 
+TEST(type_prop, tricky_label_propagation_for_auto_batch_case) {
+    auto shape = ov::PartialShape({1, 1280, 1, 1});
+    ov::DimensionTracker::set_label(shape[0], 1);
```

Comment:
If labels will be set on dimensions e.g. 0 and 2 (both 1), which label will passed to output, or will it be lost?